méli-mélo/2023-09-menu/index.html

```diff
@@ -0,0 +1,61 @@
+---
+title: Campaign Menu
+dateModified: 2023-09-11
+lang: en
+css: campaign-menu.css
+script: campaign-menu.js
+layout: no-container
+nomenu: true
+---
+        <div class="container">
+        <h1>Main content</h1>
+        <p>Example of a new menu that uses megamenu for md and lg, and GCWeb menu for sm and xs.</p>
+    </div>
+    <div style="background:#003c77;">
```

Comment:
That inline style can create some interference with the feature in a way it can provide false-positive result when assessing for WCAG conformance. 

```suggestion
    <div>
```